### PR TITLE
Ajc adjust star to support mixed genomes 517

### PR DESCRIPTION
**Pull request by ambrosejcarr**
Minor adjustments to the TagGeneExon task to make it run on larger genomes (memory usage scales with genome size)

### CODE REVIEW

File: library/tasks/StarAlignBamSingleEnd.wdl [hunk 1]
```diff
@@ -8,6 +8,7 @@ task StarAlignBamSingleEnd {
   Int input_size = ceil(size(bam_input, "G"))
   Int reference_size = ceil(size(tar_star_reference, "G"))
   Int estimated_disk_required =  ceil(input_size * 2.2 + reference_size * 2)
+  Int estimated_memory_required = reference_size + 6
```
Comment (rexwangcc) [inline library/tasks/StarAlignBamSingleEnd.wdl]: Which is larger? 30 GB or `reference_size + 6` GB?

**Comment by rexwangcc** [comment]
You may want to update this branch so that you could pass the Circle CI test.

File: library/tasks/StarAlignBamSingleEnd.wdl (hunk 1 above)
Comment (ambrosejcarr) [reply library/tasks/StarAlignBamSingleEnd.wdl]: Depends on reference_size. :)

**Comment by mckinsel** [comment]
Hey I've got a somewhat meta-question about tuning these disk size parameters. It seems like we're being pretty conservative when setting disk size, trying to cut it relatively close. But I had thought that cost of disk is pretty low, and the cost of having an engineer investigate a job that failed because it ran out of disk is very high. Does it make sense for us to ease up a little on the disk sizes? It might matter a lot once we start seeing production volumes.

**Comment by ambrosejcarr** [comment]
So I think you're right, but incidentally we managed to successfully run an absolutely huge dataset of 25x hiseqs in a single bundle, so maybe we already accomplished this (?)

... this particular example is a weird case where we had a _memory_ scaling issue that wasn't expected because memory usage scales with genome size.

**Pull request merged**